src/overlays/actors/ovl_Nameplate/ac_nameplate.c

```diff
@@ -2,33 +2,77 @@
 #include "m_actor_dlftbls.h"
 #include "m_object.h"
 #include "overlays/gamestates/ovl_play/m_play.h"
+#include "m_field_info.h"
+#include "m_player_lib.h"
+#include "m_npc.h"
+#include "m_msg_main.h"
+#include "69E2C0.h"
+#include "overlays/actors/player_actor/m_player.h"
 
-void func_80A963C0_jp(Actor* thisx, Game_Play* game_play);
-void func_80A963D0_jp(Actor* thisx, Game_Play* game_play);
+void aNP_actor_ct(Actor* thisx, Game_Play* game_play);
+void aNP_actor_dt(Actor* thisx, Game_Play* game_play);
+void aNP_set_talk_info(Actor* thisx);
+void aNP_actor_move(Actor* thisx, Game_Play* game_play);
 void aNP_actor_init(Actor* thisx, Game_Play* game_play);
 
-#if 0
 ActorProfile Nameplate_Profile = {
     /* */ ACTOR_NAMEPLATE,
     /* */ ACTOR_PART_0,
     /* */ 0,
     /* */ 0xA012,
     /* */ GAMEPLAY_KEEP,
     /* */ sizeof(Nameplate),
-    /* */ func_80A963C0_jp,
-    /* */ func_80A963D0_jp,
+    /* */ aNP_actor_ct,
+    /* */ aNP_actor_dt,
     /* */ aNP_actor_init,
     /* */ NULL,
     /* */ NULL,
 };
-#endif
 
-#pragma GLOBAL_ASM("asm/jp/nonmatchings/overlays/actors/ovl_Nameplate/ac_nameplate/func_80A963C0_jp.s")
+void aNP_actor_ct(Actor* thisx, Game_Play* game_play) {
+}
 
-#pragma GLOBAL_ASM("asm/jp/nonmatchings/overlays/actors/ovl_Nameplate/ac_nameplate/func_80A963D0_jp.s")
+void aNP_actor_dt(Actor* thisx, Game_Play* game_play) {
+}
 
-#pragma GLOBAL_ASM("asm/jp/nonmatchings/overlays/actors/ovl_Nameplate/ac_nameplate/func_80A963E0_jp.s")
+void aNP_set_talk_info(Actor* thisx) {
+    Color_RGBA8 color;
+    s32 pad; // TODO: find out what this pad is/if it's part of sp1C
+    s32 sp1C;
 
-#pragma GLOBAL_ASM("asm/jp/nonmatchings/overlays/actors/ovl_Nameplate/ac_nameplate/func_80A9648C_jp.s")
+    mNpc_GetAnimalPlateName(&sp1C, thisx->world.pos);
+    mMsg_Set_free_str(mMsg_Get_base_window_p(), 0, &sp1C, 6);
```

Comment:
From gc, 1C is pretty much the string pointer, so it's probably char [] instead of 1C and a pad

Reply:
hm makes sense, its probably the name of the villager that owns the house 
can i name it like `villagerName` or `ownerName` or something?

Reply:
Probably villager better since it's only for villagers